rdf_differ/skos_history_wrapper.py

```diff
@@ -111,9 +117,9 @@ def query_uri(self) -> str:
 
     @staticmethod
     def get_file_format(file: str) -> str:
-        file_format = guess_format(file, INPUT_MIME_TYPES)
+        file_format = guess_format(str(file), INPUT_MIME_TYPES)
         if file_format is None:
-            raise Exception('Format of "{}" is not supported.'.format(file))
+            raise SKOSException('Format of "{}" is not supported.'.format(file))
```

Comment:
Here, I would  `raise ValueError('Format of "{}" is not supported.'.format(file))`
Or maybe TypeError